### PR TITLE
For #11. Tests added for getFileType in Utility class.

### PR DESCRIPTION
**Pull request by eywoon**
Has 100% coverage currently.

### CODE REVIEW

File: app/src/main/java/us/shandian/giga/util/Utility.java [hunk 1]
```diff
@@ -126,15 +128,43 @@ public static String getFileExt(String url) {
...
+
+       if(file.endsWith(".mp3")) cc.visitBranch(0, data); //0
+       else if(file.endsWith(".wav")) cc.visitBranch(1, data); //1
+       else if(file.endsWith(".flac")) cc.visitBranch(2, data); //2
+       else if(file.endsWith(".m4a")) cc.visitBranch(3, data); //3
+
+       if (file.endsWith(".mp3") || file.endsWith(".wav") || file.endsWith(".flac") || file.endsWith(".m4a")) {  // (0||1||2||3)
+           return FileType.MUSIC;
+       }
+       else cc.visitBranch(4, data); //4
```
Comment (emillind) [inline app/src/main/java/us/shandian/giga/util/Utility.java]: Remove this, it is not a branch

**Review by emillind — CHANGES_REQUESTED** [review]
Looks good except one branch too much

File: app/src/test/java/Assignment4/GroupedTests.java [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 import org.junit.Test;
 
+import us.shandian.giga.CodeCoverage;
```
Comment (joakimboman) [inline app/src/test/java/Assignment4/GroupedTests.java]: This import does not work for me, shouldnt it import Assignment4.CodeCoverage?

File: app/src/main/java/us/shandian/giga/util/Utility.java [hunk 3]
```diff
@@ -19,6 +19,8 @@
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
+import us.shandian.giga.CodeCoverage;
```
Comment (joakimboman) [inline app/src/main/java/us/shandian/giga/util/Utility.java]: This imoport also does not work for me, import Assignment4.CodeCoverage?

**Review by joakimboman — APPROVED** [review]
Works now

**Review by emillind — APPROVED** [review]

**Pull request merged**